<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inscription</class>
 <widget class="QDialog" name="inscription">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="accessibleName">
   <string>Inscription</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frameInscription">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(51, 51, 51);
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonInscription">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>560</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);
background-color: rgb(26, 82, 118);
border-radius: 60px;
</string>
      </property>
      <property name="text">
       <string>S'inscrire</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelInscription">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 21pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Inscription</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNom">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border: none;
border-bottom: 1px solid #717072;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Nom....</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPrenom">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border: none;
border-bottom: 1px solid #717072;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Prénom...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPseudo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border: none;
border-bottom: 1px solid #717072;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Pseudo...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEmail">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>260</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border: none;
border-bottom: 1px solid #717072;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhEmailCharactersOnly</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Email...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMotDePasse">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border: none;
border-bottom: 1px solid #717072;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Mot de Passe...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRMotDePasse">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border: none;
border-bottom: 1px solid #717072;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Répeter mot de passe...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonInscriptionConnexion">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>650</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);
background-color: rgb(26, 82, 118);
border-radius: 60px;
</string>
      </property>
      <property name="text">
       <string>Se connecter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>615</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 20pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>OU</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
